odoo/addons/base/views/res_partner_views.xml

```diff
@@ -218,7 +213,7 @@
                             <field name="mobile" widget="phone"/>
                             <field name="user_ids" invisible="1"/>
                             <field name="email" widget="email" context="{'gravatar_image': True}" attrs="{'required': [('user_ids','!=', [])]}"/>
-                            <field name="website" widget="url" placeholder="e.g. https://www.odoo.com"/>
+                            <field name="website" string="Website" widget="url" placeholder="e.g. https://www.odoo.com"/>
```

Comment:
same question + a bonus: shouldn't we consider to change the label on the field itself if we want to change its label in every view ? (okay I have no idea if that's in all views, but at least in 2 of them :p )

Reply:
Same answer without a bonus